### PR TITLE
Modified AUTO_DETECT using java.vm.name

### PR DESCRIPTION
**Pull request by Yuehan-Lin**
[ci skip]
Signed-off-by: Yuehan-Lin <Yuehan.Lin@ibm.com>

### CODE REVIEW

**Comment by smlambert** [comment]
Let's hold off with PR until we see how 
https://github.com/AdoptOpenJDK/openjdk-build/issues/942 and
https://github.com/eclipse/openj9/issues/5002
get resolved, as they may end up changing vendor to "Eclipse OpenJ9", in which case we don't need to modify the AUTO_DETECT implementation.

File: test/TestConfig/src/org/openj9/envInfo/JavaInfo.java [hunk 1]
```diff
@@ -99,14 +99,14 @@ public int getJDKVersion() {
...
         impl = impl.toLowerCase();
         if (impl.contains("ibm")) {
             return "ibm";
         } else if (impl.contains("openj9")) {
             return "openj9";
-        } else if (impl.contains("oracle") || impl.contains("hotspot") || impl.contains("adoptopenjdk")) {
+        } else if (impl.contains("oracle") || impl.contains("hotspot") || impl.contains("openjdk")) {
             return "hotspot";
         } else {
             System.out.println("Cannot determine System.getProperty('java.vm.vendor')=" + impl + "\n");
```
Comment (llxia) [inline test/TestConfig/src/org/openj9/envInfo/JavaInfo.java]: Please update the print message to `System.getProperty('java.vm.name')`

**Comment by llxia** [comment]
We are going to proceed with this change. @Yuehan-Lin Could you also rebase your branch, so we can run PR builds. Thanks.

**Comment by llxia** [comment]
jenkins test sanity all jdk8

**Comment by llxia** [comment]
jenkins test sanity all jdk11

**Comment by llxia** [comment]
aix and plinux build failed due to `ERROR: Error fetching remote repo 'origin'`

**Pull request merged**